packages/DataGrid/src/components/RowIndicator/RowIndicator.styles.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const RowContainer = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  justify-content: flex-start;
+  padding: 0;
+  width: 100%;
+`;
+
+export const RowIndexText = styled.div.attrs(({ hasFourDigitsOrMore }) => {
+  const style = hasFourDigitsOrMore
+    ? {
+        fontSize: "11px",
+        padding: "4px",
+      }
+    : {
+        fontSize: "13px",
+        padding: "6px",
+      };
+
+  return {
+    style,
+  };
+})`
+  color: ${tokens.color.blackLighten40};
+  display: inline-block;
+`;
+
+export const Checkbox = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  padding: 0;
+  width: 100%;
+`;
+
+export const Void = styled.div`
+  border: 1px solid ${tokens.color.blackLighten50};
+  border-radius: 50%;
+  height: 6px;
+  width: 6px;
```

Comment:
tokens

Reply:
I'm not 100% sure about the exact size for this, I think we will have a second-round for polishing the UI.